tests/netebpfext_unit/netebpf_ext_helper.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
+#include "..\libs\store_helper\user\ebpf_registry_helper_um.h"
```

Comment:
Ok for now but is it possible to use ebpf_store_helper.h instead of ebpf_registry_helper_um.h?

Reply:
Actually, this is not correct and *should* now be `ebpf_store_helper.h `, thanks for the catch!